ERROR - 2022-05-31 20:22:51 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2022-05-31 20:22:51 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\Login.php(17): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->save()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-05-31 20:22:55 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2022-05-31 20:22:55 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\Login.php(17): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->save()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-05-31 20:22:59 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2022-05-31 20:22:59 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\Login.php(17): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->save()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-05-31 20:23:08 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2022-05-31 20:23:08 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\Login.php(17): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->save()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 20:32:58 --> Cannot declare class App\Models\User, because the name is already in use
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-05-31 20:33:15 --> You must set the database table to be used with your query.
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(545): CodeIgniter\Database\BaseConnection->table(NULL)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(176): CodeIgniter\Model->builder()
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(14): CodeIgniter\BaseModel->findAll()
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->index()
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-05-31 20:35:54 --> You must set the database table to be used with your query.
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(545): CodeIgniter\Database\BaseConnection->table(NULL)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Model.php(176): CodeIgniter\Model->builder()
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(14): CodeIgniter\BaseModel->findAll()
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->index()
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-05-31 20:59:21 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\CvView.php(164): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 164, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('CvView', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(14): view('CvView', Array)
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->index()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 21:00:06 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\CvView.php(164): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 164, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('CvView', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(14): view('CvView', Array)
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->index()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 22:25:51 --> Invalid file: EditCv.php
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('EditCv.php')
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('EditCv', Array, true)
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(72): view('EditCv', Array)
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->Editcv('1')
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-05-31 22:26:51 --> Invalid file: EditCv.php
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('EditCv.php')
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('EditCv', Array, true)
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(72): view('EditCv', Array)
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->Editcv('4')
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-05-31 22:55:49 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\CvView.php(218): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 218, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('CvView', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(14): view('CvView', Array)
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->index()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 22:55:52 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\CvView.php(218): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 218, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('CvView', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(14): view('CvView', Array)
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->index()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 22:55:54 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\CvView.php(218): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 218, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('CvView', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(29): view('CvView', Array)
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->Search()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 22:55:55 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\CvView.php(218): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 218, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('CvView', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(14): view('CvView', Array)
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->index()
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 22:57:46 --> Invalid file: PrintCv.php
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('PrintCv.php')
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('PrintCv', Array, true)
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(94): view('PrintCv')
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->PrintCv('1')
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-05-31 22:58:46 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\PrintCv.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 23, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('PrintCv', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(94): view('PrintCv')
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->PrintCv('1')
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 22:59:16 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\PrintCv.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 22, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('PrintCv', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(94): view('PrintCv')
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->PrintCv('1')
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 22:59:24 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\PrintCv.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 22, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('PrintCv', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(94): view('PrintCv')
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->PrintCv('1')
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-05-31 23:15:24 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Views\PrintCv.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('PrintCv', Array, true)
#4 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\app\Controllers\CV.php(94): view('PrintCv')
#5 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\CV->PrintCv('1')
#6 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CV))
#7 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\TB1-Perpustakaan-PWeb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
